packages/front-end/pages/power-calculator/index.tsx

```diff
@@ -115,6 +115,16 @@ const PowerCalculationPage = (): React.ReactElement => {
         <PowerCalculationSettingsModal
           close={() => setShowModal(false)}
           onSuccess={(p) => {
+            track("power-calculation-settings-update", {
+              numberOfMetrics: p.metrics.length,
```

Comment:
There's already a `numMetrics` used for tracking snapshot analyses. Maybe lets change the name to that (you can ctrl-f for it in another `track()` call to confirm).